libpod/runtime.go

```diff
@@ -347,3 +347,13 @@ func (r *Runtime) Info() ([]InfoData, error) {
 
 	return info, nil
 }
+
+// MountContainerImage Mounts the container image rootfs and returns the mountpoint
```

Comment:
Do we need these anymore? I think the code that used them was removed.

Reply:
Removed